src/core/Radio/index.stories.tsx

```diff
@@ -1,77 +1,117 @@
 
-const actions = {
-  onChange: action("onChange"),
+const DefaultDemo = (props: Args): JSX.Element => {
+  const { text } = props;
+
+  return (
+    <RadioGroup
+      aria-labelledby="demo-radio-buttons-group-label"
+      defaultValue="demo"
+      name="radio-buttons-group"
+    >
+      <FormControlLabel
+        control={<RadioButton stage="checked" />}
+        label={text}
+        value="demo"
+      />
+    </RadioGroup>
+  );
 };
 
-const storyColumn = {
-  alignItems: "flex-start",
-  display: "flex",
-  flexDirection: "column",
+export default {
+  component: DefaultDemo,
+  title: "Radio Button",
 };
 
-storiesOf("Radio Button", module).add("unchecked", () => (
-  <div style={storyColumn as React.CSSProperties}>
-    <p>
-      <strong>Accessibility</strong>: All radio buttons should have a label.
-      This can be done with a visible form label or with the aria-label
-      attribute. Note aria labels should be meaningful based on your content.{" "}
-      <br /> Good: &ldquo;XYZ Gene&ldquo;
-      <br /> Bad: &ldquo;Unchecked&ldquo; <br />{" "}
-      https://mui.com/material-ui/react-radio-button/#accessibility
-    </p>
-    <FormControlLabel
-      control={<RadioButton onChange={actions.onChange} stage="unchecked" />}
-      label="Unchecked"
-    />
-    <FormControlLabel
-      control={
-        <RadioButton disabled onChange={actions.onChange} stage="unchecked" />
-      }
-      label="Unchecked & disabled"
-    />
-    With aria labels:
-    <RadioButton
-      inputProps={{ "aria-label": "radio button unchecked" }}
-      onChange={actions.onChange}
-      stage="unchecked"
-    />
-    <RadioButton
-      disabled
-      inputProps={{ "aria-label": "radio button unchecked and disabled" }}
-      onChange={actions.onChange}
-      stage="unchecked"
-    />
-  </div>
-));
+const DefaultTemplate: Story = (args) => <DefaultDemo {...args} />;
+
+export const Default = DefaultTemplate.bind({});
+Default.parameters = {
+  snapshot: {
+    skip: true,
+  },
+};
+
+Default.args = {
+  disabled: false,
+  text: "Label",
+};
 
-storiesOf("Radio Button", module).add("checked", () => (
-  <div style={storyColumn as React.CSSProperties}>
-    <FormControlLabel
-      control={<RadioButton onChange={actions.onChange} stage="checked" />}
-      label="Checked"
-    />
-    <FormControlLabel
-      control={
-        <RadioButton disabled onChange={actions.onChange} stage="checked" />
-      }
-      label="Checked & disabled"
-    />
-    With aria labels:
-    <RadioButton
-      inputProps={{ "aria-label": "checked" }}
-      onChange={actions.onChange}
-      stage="checked"
-    />
-    <RadioButton
-      disabled
-      inputProps={{ "aria-label": "checked and disabled" }}
-      onChange={actions.onChange}
-      stage="checked"
-    />
-  </div>
-));
+const LivePreviewDemo = (props: Args): JSX.Element => {
+  const { text } = props;
+
+  const [checked, setChecked] = React.useState([true, false]);
```

Comment:
You have initialized the state with a tuple of `[boolean, boolean]`, but down there, in the `handleCheck` functions, you are setting an array of three booleans to the state!